packages/frontend/src/app/modules/flow-builder/service/action-meta.service.ts

```diff
@@ -80,6 +82,9 @@ export class ActionMetaService {
     const { piecesManifest, release } = params;
 
     return piecesManifest.filter((piece) => {
+      if (params.environment === ApEnvironment.DEVELOPMENT) {
+        return true;
+      }
```

Comment:
any reason for this?

Reply:
Yes, If i put minimum release for next version, It will stop showing up.

It's either we bump package.json after each release, which cause the notification appears for everyone or just stop the filter in local environment.